freshman/dApp.md

```diff
@@ -1 +1,14 @@
 # dApp
+
+### Q1.
+- Hey when I do the first dApp, I have this error in my console : "Uncaught ReferenceError: ethers is not defined" is it related to this :" 'ethereum.enable()' is deprecated and may be removed in the future. Please use the 'eth_requestAccounts' RPC method instead." ?
```

Comment:
Formatting is wrong here as well, check how its done in other files

Reply:
Will fix with a new PR soon!
Thanks!